src/Provider/Azure.php

```diff
@@ -282,17 +282,14 @@ public function request($method, $ref, &$accessToken, $options = [])
         if (false !== filter_var($ref, FILTER_VALIDATE_URL)) {
             $url = $ref;
         } else {
-            if (false !== strpos($this->urlAPI, 'graph.windows.net')) {
+            if (false !== strpos($this->urlAPI, 'graph.microsoft.com')) {
```

Comment:
I would prefer if this entire condition got removed from the code. Afaik Microsoft Graph doesn't support URLs like "https://graph.microsoft.com/v1.0/<guid>/me` etc. This was for Windows Azure Graph eg. https://learn.microsoft.com/en-us/graph/migrate-azure-ad-graph-request-differences#basic-requests